tests/unit/database/test_download.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for download module."""
+from datetime import datetime
+
+import pytest
+import responses
+
+from bluesearch.database.download import (
+    download_pmc_articles,
+    get_days_list,
+    get_pmc_urls,
+)
+
+
+def test_get_days_list():
+    start_date = datetime.strptime("2021-11", "%Y-%m")
+    end_date = datetime.strptime("2021-12", "%Y-%m")
+    days_list = get_days_list(start_date, end_date)
+    assert isinstance(days_list, list)
+    assert len(days_list) == 31  # 30 days in November + 1 in December
```

Comment:
Yeh.

```python
>>> datetime.strptime("2021-12", "%Y-%m")
datetime.datetime(2021, 12, 1, 0, 0)
```

Reply:
Does it make sense?  Hmmm.....the function `get_days_list` works with a `timedelta` of a day so I guess it does make sense?

Reply:
I guess the fact that it does include the end day is useful since outside of tests the `end_date` is always going to be `datetime.today()`.

Reply:
Was just thinking that ranges are usually exclusive on the right boundary. With this we'll always re-download and overwrite the archives for the first of the month, right?

Otherwise we could think about
```python
if end_date is None:
    end_date = today + 1
```

Reply:
Agree, that it is not pythonic to include right boundaries. However, IMO as long as it is clearly documented in the docstring it is fine. I stressed it even more in 0b4d178ed83a1db175a7d42871f2bd42055600a1.

However, feel free to disagree.

Reply:
fair enough :) thanks for the doc update